### ISSUE TITLE
Value definition is empty

### ISSUE THREAD

**Issue by FlTr** [ApiGen/ApiGen#576]
```
> apigen generate --debug
Scanning sources and parsing

Value definition is empty.
```

Don't know how to interpret this message (v4.0.0)

**Comment by TomasVotruba** [comment]
Could you show me your `apigen.neon`?

**Comment by FlTr** [comment]
Of course:
```
extensions: [php]
source: [./web/backend]
destination: api/php
charset: [UTF-8]
main: lti
title: LTI Backend API
baseUrl: http://example.org/lti/api/
templateTheme: bootstrap
groups: auto
accessLevels: [public, protected, private]
php: false
sourceCode: true
tree: true
deprecated: true
todo: true
download: false
```

**Comment by TomasVotruba** [comment]
I guess this is related to the code. Could you please identify the specific `*.php` file that is causing this?

**Comment by TomasVotruba** [comment]
Ping @FlTr

**Comment by TomasVotruba** [comment]
Also ApiGen version please.

**Comment by FlTr** [comment]
Sorry for late answer. Version was 4.0, i think. Also i think you're right with your guess. There was a method with just a 
```php
/**
 *
 */
```
-block in front of it. As far as i remember, deleting it or fill in some characters was the solution.

**Comment by TomasVotruba** [comment]
Yea, empty blocks have no use.
Maybe improving exception message would help, if you find it. It might be located in [PHPTokenReflection](https://github.com/Andrewsville/PHP-Token-Reflection)

Closing then.

**Comment by nichtich** [comment]
I'd reopen until fixed with a better exception message.

**Comment by kix** [comment]
Confirmed, looks like I've stumbled upon the same issue and I don't really have any empty comment blocks.

**Comment by mihaikelemen** [comment]
The same for me (using version 4.1)

**Comment by populov** [comment]
In my case, looks like this error is caused by constant array declaration:
```php
const MYARRAY = [0, 2, 5];
```

**Comment by juvenn** [comment]
I can also confirm @populov's case,  a constant with array will cause problem.

**Comment by adamcornforth** [comment]
+1 @populov, a constant array in my project caused this problem too. 

Editing my `apigen.conf` to exclude the file with the constant array fixed the problem, e.g:

```
exclude: 
    - 'ClassWithConstantArray.php'
```

**Comment by nelson6e65** [comment]
> @populov:
> In my case, looks like this error is caused by constant array declaration
:+1:

@TomasVotruba I'm still having this issue too, caused by an array `const`.  Why is this closed? 

ApiGen: `4.1.0`
PHP: `5.6.28`

**Comment by kurinniy** [comment]
I confirm this problem with array `const`

PHP 5.6.27
ApiGen 4.1.0

File that causes problem looks like this:

```
<?php

namespace Direct;

class Constants
{
	const CLIENT_PREFIX = 'zzcli';

	const CAMPAIGN_PREFIXES = ["asdf", "zxcv", "qwer"];
}
```

**Comment by ek9** [comment]
For anyone stumbling upon this issue, the support for this will be implemented as part of #774 issue.
